Gemfile

```diff
@@ -107,6 +107,7 @@ group :test, :development do
   gem "dotenv-rails"
   gem "erb_lint", require: false
   gem "factory_bot_rails"
+  gem "faker"
```

Comment:
Just a heads up: PR #92 also adds Faker. So depending on who merges first, there _may_ be a merge conflict here.

Reply:
It's a race!